pandas/core/frame.py

```diff
@@ -1635,13 +1635,11 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
 
         Examples
         --------
-        >>> writer = StataWriter('./data_file.dta', data)
-        >>> writer.write_file()
```

Comment:
I wouldn't necessarily change these.  Perhaps just add another example that doesn't use `StataWriter`.